### PR TITLE
#87 Fix ignored tests.

### PR DESCRIPTION
**Pull request by aivinog1**
This is fix to #87. Removing ignore and fixing test.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#92](https://github.com/yegor256/cactoos-http/issues/92) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#92](https://github.com/yegor256/cactoos-http/issues/92) is assigned to @vzurauskas[/z](https://www.0crat.com/u/vzurauskas), here is [why](https://www.0crat.com/footprint/C63314D6Z/2fe24789-2bee-4ba9-a0db-c10f9b7e68f8); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 1]
```diff
@@ -76,24 +75,19 @@ public void worksFine() throws Exception {
...
-                new HtWire(
-                    this.server.getInetAddress().getHostAddress(),
-                    this.server.getLocalPort()
-                ),
-                timeout
-            ).send(new InputOf("unused"));
-        }
+        new HtTimedWire(
+            input -> {
+                TimeUnit.SECONDS.sleep(timeout + timeout);
```
Comment (vzurauskas) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 Could this be `timeout + 1` instead? I'd rather the test took as little time as possible.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 2]
```diff
@@ -76,24 +75,19 @@ public void worksFine() throws Exception {
...
+                return input;
+            },
+            timeout
```
Comment (vzurauskas) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 It's not clear whether timeout actually happens after 100 ms as specified here, or it waits full 100 + 100 ms until the function finishes execution and throws the `TimeoutException` only then. I think we need another test for that.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 3]
```diff
@@ -76,24 +75,19 @@ public void worksFine() throws Exception {
         );
     }
 
-    // @todo #78:30min With #78 solved, HtWire stopped closing the connection
-    //  when send() is called and the following test stopped working for no
-    //  clear reason. Investigate the cause of this, fix it and unignore this.
-    @Ignore("see todo above")
+    // @todo #87:30m For now I can't find proper OOP alternative for waiting
+    //  (such in this case). Needs to create a new one.
```
Comment (vzurauskas) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 Could you also describe what this alternative would need to have as opposed to the current implementation?

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 1 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Agreed. I'll fix that.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 2 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Good point, thank you! I'll add another test.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 3 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Sure. I'll add my thoughts about that in this comment.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 1 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Done.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 2 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Done.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 3 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Done.

**Comment by aivinog1** [comment]
@vzurauskas I've fix all issues that you found in your review.

**Comment by vzurauskas** [comment]
@llorllale Looks good.

**Comment by aivinog1** [comment]
@llorllale I've tried to do as much as possible but found out some issues that I've marked as todo.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 4]
```diff
@@ -45,8 +50,11 @@
  * @checkstyle JavadocVariableCheck (500 lines)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
+@SuppressWarnings("PMD.TooManyMethods")
 public final class HtTimedWireTest {
 
+    private static final String UNUSED = "UNUSED";
+    private static final String TIMEOUT = "Did not failed after timeout";
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 please remove these two constants:

* both these constants introduce hidden coupling between tests
* if `UNUSED` were really unused, it would not be needed
* `TIMEOUT` has the wrong message structure - it should affirm the behavior being tested. Eg. if you're asserting that it throws a TimedOutException, the assertion's message would be "fails with a timedoutexception"

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 5]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
 
-    // @todo #78:30min With #78 solved, HtWire stopped closing the connection
-    //  when send() is called and the following test stopped working for no
-    //  clear reason. Investigate the cause of this, fix it and unignore this.
-    @Ignore("see todo above")
-    // @checkstyle MagicNumberCheck (1 line)
-    @Test(expected = TimeoutException.class, timeout = 1000)
+    // @todo #87:30m For now I can't find proper OOP alternative for waiting.
+    //  Needs to create a new one. I think this can looks like this:
+    //  new Wait().seconds(long seconds). Needs to do this in the test
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 I'm not sure this is needed and I don't really like the proposed solution. "Wait" is an action, not a noun, hence cannot be made a proper object.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 6]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
-        try (Socket blocker = new Socket()) {
-            blocker.connect(this.server.getLocalSocketAddress());
-            new HtTimedWire(
-                new HtWire(
-                    this.server.getInetAddress().getHostAddress(),
-                    this.server.getLocalPort()
-                ),
-                timeout
-            ).send(new InputOf("unused"));
+        final HtTimedWire wire = new HtTimedWire(
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 any reason why `wire` cannot be of type `Wire` (the highest level of abstraction)?

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 7]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
+                return input;
+            },
+            timeout
+        );
+        new Assertion<>(
+            HtTimedWireTest.TIMEOUT,
+            () -> wire.send(new InputOf(HtTimedWireTest.UNUSED)),
+            new TimeoutExceptionMatcher()
+        )
+            .affirm();
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 `.affirm()` should be attached to the closing parenthesis

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 8]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
+        final HtTimedWire wire = new HtTimedWire(
+            input -> {
+                TimeUnit.SECONDS.sleep(sleep);
+                return input;
+            },
+            timeout
+        );
+        final long current = System.currentTimeMillis();
+        new Assertion<>(
+            HtTimedWireTest.TIMEOUT, () -> wire.send(
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 indentation is way off inside this `Assertion`, making it really hard to follow and group the individual components together

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 9]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
+     *  I've tried to update to the new version of matchers and
+     *  use constructor of Throws with {@link org.hamcrest.Matcher},
+     *  but it leads update the cactoos
+     *  and after that I can't build a project.
+     *  So needs to update dependencies, fix conflicts and replace
+     *  this class with {@link org.llorllale.cactoos.matchers.Throws}.
+     *  And needs try to remove {@link java.lang.SuppressWarnings} in
+     *  {@link org.cactoos.http.HtTimedWireTest}.
+     */
+    private final class TimeoutExceptionMatcher extends
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 leave a `@todo` to replace this with llorllale/cactoos-matchers#133

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 10]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
+     * all of {@link org.cactoos.http.HtTimedWireTest.AllOf#matchers}.
+     * @todo #87:30m For a some reason org.hamcrest.core.AllOf matcher with
+     *  org.cactoos.iterable.IterableOf is not working (compilation error with
+     *  unchecked generic array creation for varargs parameter).
+     *  Needs to replace this with org.hamcrest.core.AllOf
+     *  and deal with compilation error.
+     *  And needs try to remove {@link java.lang.SuppressWarnings} in
+     *  {@link org.cactoos.http.HtTimedWireTest}.
+     */
+    private final class AllOf extends TypeSafeDiagnosingMatcher<Scalar<Input>> {
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 why not use `AllOf` from hamcrest?

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 11]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
+            HtTimedWireTest.TIMEOUT,
+            () -> wire.send(new InputOf(HtTimedWireTest.UNUSED)),
+            new TimeoutExceptionMatcher()
+        )
+            .affirm();
+    }
+
+    // @checkstyle MagicNumberCheck (1 line)
+    @Test(timeout = 1000)
+    public void failsAfterTimeoutCheckIfWait() throws Exception {
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 this test is a bit hard to follow. Is the objective here to verify that `HtTimedWire` actually failed after `timeout` expires and not before?

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 12]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
+        final long timeout = 100;
+        final long sleep = 10 * timeout;
+        final HtTimedWire wire = new HtTimedWire(
+            input -> {
+                TimeUnit.SECONDS.sleep(sleep);
+                return input;
+            },
+            timeout
+        );
+        final long current = System.currentTimeMillis();
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 this variable would best be named `start` for clarity

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 13]
```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
...
+            HtTimedWireTest.TIMEOUT, () -> wire.send(
+            new InputOf(HtTimedWireTest.UNUSED)
+        ), new AllOf(
+            new TimeoutExceptionMatcher(),
+            new TypeSafeDiagnosingMatcher<Scalar<Input>>() {
+                @Override
+                protected boolean matchesSafely(
+                    final Scalar<Input> item,
+                    final Description description) {
+                    final long failed = System.currentTimeMillis();
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 no need to declare `failed`. The `return` statement below would be perfectly readable as `return System.currentTimeMillis() - start < sleep;`

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 4 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: I agree, thanks!

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 5 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Good point. I've thought about it again and decide that TimeUnit not so bad decision.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 6 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: I agree, thanks!

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 7 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Okay

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 8 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Okay, I'll try to do so. But what do you think about to configure the qulice such way, that bad formatted code doesn't pass pipeline?

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 9 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Are you sure? This matcher only matches that exception will throw not about timeouts.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 10 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Because it requires Iterable, but when I create Iterable with a generic anonymous class, then it fails. I'll fix it (replace generic anonymous class to an inner class ).

Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: And I'll add a todo that you posted here: https://github.com/yegor256/cactoos-http/pull/92#discussion_r299099594

Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: > Because it requires Iterable, but when I create Iterable with a generic anonymous class, then it fails. I'll fix it (replace generic anonymous class to an inner class ).

A quick fix cannot help me, I've added a todo.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 11 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: No. Its primary objective from this comment: https://github.com/yegor256/cactoos-http/pull/92#discussion_r296634032. So this test verifies that execution of sending will be interrupted and we don't wait for executing of the origin in the HtTimedWire.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 12 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: I agree

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 13 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: I agree.

**Comment by aivinog1** [comment]
@llorllale I've fixed or answered about all you review comments.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 8 above)
Comment (llorllale) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 sounds like a bug for qulice

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 9 above)
Comment (llorllale) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 oh I see

Comment (llorllale) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 this means you can use `Throws` from cactoos-matchers

Comment (llorllale) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 please rename to `ThrowsTimeoutException`

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 14]
```diff
@@ -76,24 +84,131 @@ public void worksFine() throws Exception {
...
+        final long sleep = 10 * timeout;
+        final Wire wire = new HtTimedWire(
+            input -> {
+                TimeUnit.SECONDS.sleep(sleep);
+                return input;
+            },
+            timeout
+        );
+        final long start = System.currentTimeMillis();
+        final JoinedText message = new JoinedText(
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 the assertion's message should express a single assertion. Asserting two things means you're testing two things at the same time

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 15]
```diff
@@ -76,24 +84,131 @@ public void worksFine() throws Exception {
...
+     *  https://github.com/llorllale/cactoos-matchers/issues/133
+     *  when an issue will be resolved. This is a temporary solution
+     *  for asserting a maximum time.
+     */
+    private final class MaximumTimeMatcher
+        extends TypeSafeDiagnosingMatcher<Scalar<Input>> {
+        private final long start;
+        private final long sleep;
+
+        MaximumTimeMatcher(final long start, final long sleep) {
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 `sleep` would best be renamed as `duration`

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 16]
```diff
@@ -76,24 +84,131 @@ public void worksFine() throws Exception {
...
+            " fails with timeout exception ",
+            " execution isn't interrupted when a timeout is exceeded "
+        );
+        new Assertion<>(
+            message.asString(),
+            () -> wire.send(new InputOf()),
+            new AllOf<>(
+                new IterableOf<>(
+                    new TimeoutExceptionMatcher(),
+                    new MaximumTimeMatcher(start, sleep)
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 I think you should be passing in `timeout` instead of `sleep` to `MaximumTimeMatcher`

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 14 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: I agree.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 15 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: Okay

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 9 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: @llorllale Done.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java (hunk 16 above)
Comment (aivinog1) [reply src/test/java/org/cactoos/http/HtTimedWireTest.java]: @llorllale The time of execution is slightly more than the timeout, but less than sleep. So, the condition in the matcher is less than (strict).

**Comment by aivinog1** [comment]
@llorllale I've fixed or answered to all your review comments.

File: src/test/java/org/cactoos/http/HtTimedWireTest.java [hunk 17]
```diff
@@ -76,24 +83,126 @@ public void worksFine() throws Exception {
...
+        final Wire wire = new HtTimedWire(
+            input -> {
+                TimeUnit.SECONDS.sleep(sleep);
+                return input;
+            },
+            timeout
+        );
+        final long start = System.currentTimeMillis();
+        new Assertion<>(
+            "Execution isn't interrupted when a timeout is exceeded",
```
Comment (llorllale) [inline src/test/java/org/cactoos/http/HtTimedWireTest.java]: @aivinog1 is this asserting that `InterruptedException` is not thrown?

**Comment by victornoel** [comment]
@llorllale I've made #93 that solves the same ticket (I was assigned by 0crat, I suppose @aivinog1 left the project) in a much simpler way.

**Comment by llorllale** [comment]
@victornoel thanks. This PR was incredibly challenging to understand for me.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@dddff4c1-9156-49e2-a2d9-cf978c5a9655, thread: pool-11-thread-1
com.zerocracy.farm.sync.SyncProject[76] java.lang.IllegalStateException: Failed to acquire "wbs.xml" in "C63314D6Z" in 2min: java.util.concurrent.locks.ReentrantLock@6d09edc9[Locked by thread pool-470-thread-1]
```

[0.48.2](https://github.com/zerocracy/farm): CID: [e7862061-1753-44a6-9450-7763d1e295c3](https://www.0crat.com/footprint/C63314D6Z/e7862061-1753-44a6-9450-7763d1e295c3), Type: "Close job"

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@dddff4c1-9156-49e2-a2d9-cf978c5a9655, thread: pool-11-thread-1
com.zerocracy.farm.sync.SyncProject[76] java.lang.IllegalStateException: Failed to acquire "orders.xml" in "C63314D6Z" in 2min: java.util.concurrent.locks.ReentrantLock@6d09edc9[Locked by thread pool-470-thread-1]
```

[0.48.2](https://github.com/zerocracy/farm): CID: [e7862061-1753-44a6-9450-7763d1e295c3](https://www.0crat.com/footprint/C63314D6Z/e7862061-1753-44a6-9450-7763d1e295c3), Type: "Close job"

**Comment by vzurauskas** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos-http/issues/92#issuecomment-513875130))

@vzurauskas This is what I know about this job in [C63314D6Z](https://www.0crat.com/p/C63314D6Z), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#92](https://github.com/yegor256/cactoos-http/issues/92) is [in scope](http://www.zerocracy.com/policy.html#14) for 28days
  * The role is `REV`
  * The job is assigned to @vzurauskas[/z](https://www.0crat.com/u/vzurauskas) for [28days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @aivinog1[/z](https://www.0crat.com/u/aivinog1): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos-http%2392%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by vzurauskas** [comment]
@0crat out

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vzurauskas[/z](https://www.0crat.com/u/vzurauskas), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Code review was too long (28 days), architects (@llorllale) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/cactoos-http/issues/92#issuecomment-513875884))

@vzurauskas The job [#92](https://github.com/yegor256/cactoos-http/issues/92) is now out of scope

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/cactoos-http/issues/92#issuecomment-513875884))

@vzurauskas Pull request [#92](https://github.com/yegor256/cactoos-http/issues/92) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by sereshqua** [comment]
@aivinog1 please make sure you start all your messages with the name of the user they are addressed to, [see](https://github.com/yegor256/cactoos-http/pull/92#discussion_r296645423) for instance

**Comment by sereshqua** [comment]
@aivinog1 ping

**Comment by aivinog1** [comment]
@sereshqua Done.

**Comment by sereshqua** [comment]
@aivinog1 thanks! that was just 1 message, but you have many more, next time plz always start all messages in jobs / PRs from the name of the user

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @vzurauskas[/z](https://www.0crat.com/u/vzurauskas)
